packages/EasyCore/src/Bridge/Laravel/Providers/EasyCoreServiceProvider.php

```diff
@@ -44,4 +47,18 @@ private function logQueueWorkerStopping(): void
 
         $this->app->get('events')->listen(WorkerStopping::class, QueueWorkerStoppingListener::class);
     }
+
+    /**
+     * Restarts Queue.
+     *
+     * @return void
+     */
+    private function restartQueue(): void
```

Comment:
Please rename to `restartQueueOnEmClose` and remove the method description (to make the code similar to the existing one).

Reply:
Done.